tox.ini

```diff
@@ -40,7 +40,7 @@ basepython =
     python2.7
 commands =
     coverage run -m zope.testrunner --test-path=. --auto-color --auto-progress []
-    coverage report
+    coverage report --fail-under=92
```

Comment:
It looks like this number can be bumped to 96% based on the coveralls.io reports.

Reply:
I got only 92 % using tox when using the C implementation. When I switched to to use the pure Python one it even fell to 89 %. So the combined coverage seems to have more impact than it seems. But I did not add combined coverage to to `tox.ini` as the tests are already slow.